examples/configs/sensors/mbient_acc_1.yml

```diff
@@ -0,0 +1,4 @@
+sensor_id: mbient_acc_1
+temporal_res: 100.0
+file_type: xdf
```

Comment:
Since everything will (at least in part) stream to an XDF, should we perhaps only include a file type when there is an additional raw data file beyond what we send to LSL?